src/main/webapp/forms/formUtilities.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Sets one form input's value to the given value.
+ */
+function setFormInputValue(input, value) {
+  if (value === undefined) {
+    input.value = '';
+  } else {
+    input.value = value;
+  }
+}
+
+/**
+ * This function handles an event of a file input
+ * specified by the id @param fileInputName.
+ * If a new file is uploaded, @param fileUploaded function is called
+ * with the file's name as parameter.
+ * Otherwise, fileRemovedFunction is called.
+ */
+function handleFileInputChangeEvent(fileInputName,
+                                    fileUploadedFunction,
+                                    fileRemovedFunction) {
+  const fileInput = document.getElementById(fileInputName);
+  fileInput.addEventListener("change", function () {
+    const file = this.files[0];
+    if (file) {
+      const reader = new FileReader();
+      reader.addEventListener("load", function () {
+        fileUploadedFunction(this.result);
+      });
+      reader.readAsDataURL(file);
+    } else {
+      fileRemovedFunction();
+    }
+  });
+}
+
+/**
+ * Whenever the clear-img-icon button is clicked, the current photo is removed,
+ * meaning that it won't be submitted
+ * (or, if the ucrrent photo is the previously saved photo,
+ * then it won't be kept)
+ * with the next submission.
+ * This change will be reflected in the preview too.
+ */
+function activateRemoveImageFeature(hasPreviousImage) {
```

Comment:
How about passing the element id as a parameter, like you did above for the files as well. That way this function would work with multiple elements.

Reply:
Done.